docs/learn/dapp-developer/readData.md

```diff
@@ -94,12 +84,13 @@ async function fetchProfile(address) {
     const profile = new ERC725(erc725schema, address, provider, config);
     return await profile.fetchData();
   } catch (error) {
+    console.log(error);
```

Comment:
As discussed in our meeting, we should only use functions, try/catch, and logs in the sample implementation (LUKSO Playground)

The tech docs should focus on the main code that is used to fetch the data to keep everything simple. 

Here, it should state: `await new ERC725(erc725schema, address, provider, config).fetchData();` 
(instead of 7 lines of code)